oximeter/db/schema/replicated/db-init-1.sql

```diff
@@ -0,0 +1,204 @@
+CREATE DATABASE IF NOT EXISTS oximeter ON CLUSTER oximeter_cluster;
```

Comment:
I think it'd be helpful to add some more documentation about these two SQL files at the top here. Why are we splitting them? How do we decide what goes in each file? Related, could we have a test that ensures they remain disjoint?

Reply:
Agreed. I'll add some documentation. 

In thinking about how to add such a test, I think I would write it in the following manner:

1. Spin up a cluster (single node, single keeper for speed)
2. Run db-init-1.sql
3. Record the created tables
4. Delete all the tables or teardown and recreate cluster
5. Run db-init-2.sql
6. Record the created tables
7. Ensure that the sets of tables are disjoint

The only issue with this is that the database creation itself only occurs in `db-init-1.sql`. I'd have to also add that to `db-init-2.sql`. However, the call is idempotent, so I don't see an issue with doing that.

Reply:
Done in [c5e293f](https://github.com/oxidecomputer/omicron/pull/6131/commits/c5e293f852efcf18f0ea8ddb6610af71fb5f4499)

I also removed some tables from `db-init-1.sql` that were unused in tests to add further speedup.